pages/celebrity-details.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>

```

Comment:
Can we rename this file to `notable-person.html`? We call the celebrity details page Notable Person page.